### ISSUE TITLE
Keep cert-manger installation outside of opentelemetry-operator module

### ISSUE THREAD

**Issue by gohmc** [aws-ia/terraform-aws-eks-blueprints#757]
The [constraint](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/4d99fd7a0e52b81962c7b09dd0d80c3c54ff5bd2/modules/kubernetes-addons/opentelemetry-operator/main.tf#L6) to using cert-manager < 1.6.0 has been lifted as document [here](https://docs.aws.amazon.com/eks/latest/userguide/adot-reqts.html).  While the version can be easily updated, but the cert-manager resource in the opentelemetry-operator module does not allow any customization and will conflict with existing cert-manager configured for the underlying cluster.

Suggest do not include cert-manager installation in the opentelemetry-operator module and let the addon or operator to use the cert-manager installed with `enable_cert_manager`.  The code [here](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/4d99fd7a0e52b81962c7b09dd0d80c3c54ff5bd2/modules/kubernetes-addons/main.tf#L379) can be updated to ensure existence of cert-manager:

```
count = var.enable_amazon_eks_adot || var.enable_opentelemetry_operator ? var.enable_cert_manager ? 1 : 0 : 0
depends_on = [module.cert_manager]
...
```

**Comment by askulkarni2** [comment]
@bonclay7 can you please confirm we can remove the cert-manager version `v1.5.0` hard-coding now?

**Comment by bonclay7** [comment]
Yes, thanks for raising this. We will send an update shortly. As for reusing an existing `cert-manager`, we might take a different approach

**Comment by askulkarni2** [comment]
@bonclay7 is this issue still needed?
